base/acme/database/postgresql/create.sql

```diff
@@ -56,5 +56,6 @@ CREATE TABLE "authorization_challenges" (
 
 CREATE TABLE "certificates" (
     "id"               VARCHAR PRIMARY KEY,
-    "data"             BYTEA
+    "data"             BYTEA,
+    "expires"          TIMESTAMP
```

Comment:
I don't think we need that since we're storing UTC timestamps created with Java `Date` class. The time zone only matters when we display/parse the timestamps using Java `DateFormat`.

Reply:
Hmm.. PostgreSQL itself is treating it as a local time. Let me take a look more on this..

Reply:
Let me take care of this in a separate PR. There are other timestamp columns that might need to be fixed as well. Good catch. Thanks!